src/xrUICore/XML/UIXmlInitBase.cpp

```diff
@@ -1167,6 +1167,15 @@ bool CUIXmlInitBase::InitTrackBar(CUIXml& xml_doc, LPCSTR path, int index, CUITr
         }
     }
 
+    string512 buf;
```

Comment:
Why not xr_string?

Reply:
100% copy paste from GSC code.

Reply:
@FreeZoneMods `xr_string` has code overheads for this point of use.

Reply:
This code executes not so frequently, so, the performance decreasing shouldn't be noticeably. Using xr_string will make the code slight safer.
I think we shouldn't follow bad practices (introduced by GSC) when writing new code and get rid of such code with the development of the project.